syntax = "proto3";
package user;

service UserService {
  rpc GetUser (UserRequest) returns (UserReply);
  rpc CreateUser (CreateUserRequest) returns (UserReply);
}

message UserRequest {
  string id = 1;
}

message UserReply {
  string id = 1;
  string name = 2;
}

message CreateUserRequest {
  string name = 1;
}
